### PR TITLE
Update k8s.tlsSecretName property behaviour doc

### PR DESCRIPTION
**Pull request by mmorhun**
Signed-off-by: Mykola Morhun <mmorhun@redhat.com>

> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
Updates docs regarding `k8s.tlsSecretName` CR field property.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18079

### Specify the version of the product this PR applies to.
Since merge of https://github.com/eclipse/che-operator/pull/508

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

File: modules/installation-guide/partials/ref_checluster-custom-resource-fields-reference.adoc
Comment (yhontyk) [inline modules/installation-guide/partials/ref_checluster-custom-resource-fields-reference.adoc]:
```suggestion
`tlsSecretName`: che-tls: Name of a secret that is used to set ingress TLS termination if TLS is enabled. If the specified secret does not exist, a self-signed certificate will be created. If the value empty or omitted, the default ingress certificate will be used. See also the `tlsSupport` field. Note, if switch from user-defined to cluster default ingress certificate, `self-signed-certificate` secret should be deleted manually.
```

**Comment by mmorhun** [comment]
@boczkowska thank you for the corrections. However vale check is still failing because of words `if` and `should`. Deleting them and rewording looks artificial for me. What is the right procedure in such a case?

**Review by yhontyk — APPROVED** [review]
@mmorhun I find the use of `if` and `should` to be correct and acceptable here. I'll file an issue to tune those vale rules. 

I'll approve the PR, please merge when ready. Note that currently we publish 7.21 docs from the master branch. If it applies to the future release, we'll have to wait until the next release is GA.

**Pull request merged**
